start.sh

```diff
@@ -335,6 +335,12 @@ if [ "$load_config" == "true" ]; then
             load_from 'SPVWALLET_ADMIN_XPRIV' admin_xpriv
             load_from 'RUN_ADMIN_PANEL' admin_panel
         done < ".env.config"
+
+        if [ -n "$paymail_domain" ]; then
+            SPVWALLET_NODES_CALLBACK_CALLBACK_HOST="https://$paymail_domain"
```

Comment:
Beeing late to the party :)
It's bit redundant to have double callback in the name (I know it mirrors yaml structure). "host" is already nested inside callback - there is no need to repeat.

Reply:
Good point, but it's a topic for separate task.

Reply:
Agree

Reply:
Agree